dev/ColorPicker/ColorPicker_themeresources.xaml

```diff
@@ -28,6 +28,6 @@
     </ResourceDictionary.ThemeDictionaries>
     <Style x:Key="ColorPickerBorderStyle" TargetType="Shape">
         <Setter Property="Stroke" Value="{ThemeResource SystemControlForegroundListLowBrush}" />
-        <Setter Property="StrokeThickness" Value="2" />
+        <Setter Property="StrokeThickness" Value="1" />
```

Comment:
This only changes the "RGB" ComboBox border, the rests are unchanged. Also, I don't think we need to update border of the picker, only those ComboBoxes need to be updated.

![20190725142047](https://user-images.githubusercontent.com/4424330/61909621-872c0c00-aee7-11e9-814a-20332765f35a.png)

Reply:
The TextBoxes also need to go to 1px. TextBoxes go to 2px when the control is active but not when it is rest.

Reply:
I see, I mistakenly thought they were all ComboBoxes for some reason...
Those TextBoxes should get updated automatically once we change the default TextBox style.